### PR TITLE
new redux structure

### PR DESCRIPTION
**Pull request by riturajpandey**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = {
```
Comment (raineorshine) [inline .eslintrc.js]: Is there a reason there is an `.eslintrc.js` and an `.eslintrc.json`?

File: Appbackup.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { AppLoading } from 'expo';
```
Comment (raineorshine) [inline Appbackup.js]: I think `Appbackup.js` was used for debugging and can be deleted now?

File: actions/ActionTypes.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const ADD_THOUGHT = 'add_thought';
```
Comment (raineorshine) [inline actions/ActionTypes.js]: While unconventional, I prefer using string literals in camelCase for action types. I have not noticed any benefit to action type definitions as mistyped action types can throw in the reducer.

**Comment by riturajpandey** [comment]
I have some queries:

'I suggest having the same actions and reducers in this project as in the web project. It will be too difficult re-implementing some of the more complicated logic otherwise. That said, I understand that a lot of the new React Native functionality will require its own actions. That is okay as long as the common operations are represented by common actions and reducers'

=> I am using actions and reducers in a quite different way as that of yours. I am not able to reuse your logic in reducers. However, I can try to reuse to some extent as I can.

Another query I have: Do you want to merge your reducers folder in my code or it will be separate as it is now.

'This project conforms to the functional programming style, as indicated in the contributing guidelines and our Upwork contract. Mutations should be avoided. I will provide more detailed feedback on procedural code once we get this original PR reviewed in order to keep things simple, but I just want you reiterate that this is a strict requirement on this project.'

" Mutations should be avoided" => Did you mean avoid Mutations as in the link.
https://www.zeptobook.com/object-mutation-in-javascript/

Correct me if I wrong?

**Pull request closed without merging**